ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -672,6 +672,10 @@ def std(self, axis=None, unbiased=True, keepdim=False, name=None):
     def trunc(self, name=None):
         return paddle_frontend.Tensor(ivy.trunc(self._ivy_array))
 
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+    def stanh(self, scale_a=0.67, scale_b=1.7159, name=None):
+        return paddle_frontend.stanh(self._ivy_array, scale_a=scale_a, scale_b=scale_b)
+
```

Comment:
About the error you mentioned, the error message suggests the failure is happening in the paddle backend. So, to manually test it, you would first need to do `ivy.set_paddle_backend`. The correct value you are getting in the manual test is coming from a different backend.

Reply:
yeah i forget to set the paddle backend before i test the manually. but still it's giving the right answer in manual testing not returning the NAN value.

i thank to solve this issue we should made a stanh primary function in ivy frontend.

thank you